test/common/setTimeout.spec.ts

```diff
@@ -102,6 +102,27 @@ describe('setTimeout', function() {
     expect(typeof(cancelId.toString())).toBe('number');
   });
 
+  it('should allow cancelation by numeric timeout Id', function (done) {
+    // Node returns complex object from setTimeout, ignore this test.
+    if (isNode) {
+      done();
+      return;
+    }
+
+    const testZone = Zone.current.fork(Zone['wtfZoneSpec']).fork({ name: 'TestZone' });
+    testZone.run(() => {
+      const spy = jasmine.createSpy('spy');
+      const task: Task = <any>setTimeout(spy, 0);
+      const cancelId: number = <any>task;
+      clearTimeout(0 + cancelId);
+      setTimeout(function () {
```

Comment:
Right.

Done. Set the second timeout of 1.

Please note that the same point also applies to another test in this file as well, see https://github.com/platosha/zone.js/blob/02d0784bb9e9cfb44d2eaa32c12be538370b0a0b/test/common/setTimeout.spec.ts#L57